.stylelintrc.json

```diff
@@ -1,6 +1,44 @@
 {
+    "extends": "stylelint-config-standard-scss",
+    "files": ["**/*.scss"],
     "customSyntax": "postcss-scss",
     "rules": {
+        "indentation": 4,
+        "string-quotes": "single",
+        "declaration-empty-line-before": "never", 
+        "declaration-no-important": true, 
+        "color-function-notation": "legacy",
+        "alpha-value-notation": "number",
+        "property-no-vendor-prefix": null,
+        "value-no-vendor-prefix": null,
+        "at-rule-empty-line-before": null,
+        "no-descending-specificity": null,
+        "value-keyword-case": [
+            "lower",
+            {
+                "camelCaseSvgKeywords": true,
+                "ignoreFunctions": ["local"]
+            }
+        ],
+        "max-line-length": null,
+        "font-family-no-missing-generic-family-keyword": [
+            true,
+            {
+                "ignoreFontFamilies": ["FA-Icons-Contact"]
```

Comment:
We don't have this font in Protocol (I'm not even sure we still have it in bedrock) so we don't need this exception.